### DISCUSSION TITLE
Understand the difference between dcm4chee and dcm4chee arcieve.

### DISCUSSION THREAD

**Discussion by njiddasalifu** [dcm4che/dcm4chee-arc-light#4415] [General]
Hello, i am trying to setup a pacs server for our plartform and the choice is dcm4chee. 
Now the problem i have is what is the difference between dc4mchee and dcm4chee arcieve as all of them have different wiki docummentations and different ways of installations. I am open to learn from experience and get advice how to move on with my work. thanks

**Comment by vrindanayak** [comment]
[dcm4che](https://github.com/dcm4che/dcm4che) is [DICOM Toolkit & Library](https://github.com/dcm4che/dcm4che?tab=readme-ov-file#dcm4che-dicom-toolkit--library) whereas [dcm4chee-arc-light](https://github.com/dcm4che/dcm4chee-arc-light) is a [lightweight DICOM Archive Java EE application running in WildFly](https://github.com/dcm4che/dcm4chee-arc-light?tab=readme-ov-file#dcm4chee-archive-5x).

DCM4CHE toolkit and libraries are also used by other open source products who are active on the [dcm4che forum](https://groups.google.com/forum/#!forum/dcm4che).
DCM4CHEE Archive 5.x is used primarily for storing and querying / retrieving patient examinations (DICOM objects)

**Reply by njiddasalifu** [reply]
thanks @vrindanayak it's clear.
